indy_node/server/node.py

```diff
@@ -93,21 +88,6 @@ def __init__(self,
         # TODO: ugly line ahead, don't know how to avoid
         self.clientAuthNr = clientAuthNr or self.defaultAuthNr()
 
-        self.configLedger = self.getConfigLedger()
-        self.ledgerManager.addLedger(
-            CONFIG_LEDGER_ID,
-            self.configLedger,
-            postCatchupCompleteClbk=self.postConfigLedgerCaughtUp,
-            postTxnAddedToLedgerClbk=self.postTxnFromCatchupAddedToLedger)
-        self.on_new_ledger_added(CONFIG_LEDGER_ID)
-        self.states[CONFIG_LEDGER_ID] = self.loadConfigState()
-        self.upgrader = self.getUpgrader()
-        self.poolCfg = self.getPoolConfig()
-        self.configReqHandler = self.getConfigReqHandler()
-        self.initConfigState()
-        self.register_req_handler(CONFIG_LEDGER_ID, self.configReqHandler)
-        self.requestExecuter[CONFIG_LEDGER_ID] = self.executeConfigTxns
```

Comment:
Where do we register executors for CONFIG ledger?

Reply:
Yes